packager/packager.cc

```diff
@@ -81,7 +82,24 @@ struct Import {
     Import(PackageName &&name, ImportType type) : name(std::move(name)), type(type) {}
 };
 
-struct PackageInfo {
+class PackageInfoImpl final : public sorbet::core::packages::PackageInfo {
```

Comment:
ooc, why is the implementation a subclass of an abstract base class if (from what i can see) there's only one implementation?

Reply:
@jvilk's suggestion wanted to keep the details of the package representation (FQN, PackageName, ImportType, etc.) out of core. Similar to WorkerPoolImpl.